python/dolfinx/fem/petsc.py

```diff
@@ -112,27 +112,29 @@ def fn(form):
 
 
 def create_vector(
-    L: typing.Union[Form, Iterable[Form]], kind: typing.Optional[str] = None
+    container: typing.Union[Form, _FunctionSpace, Iterable[Form], Iterable[_FunctionSpace]],
```

Comment:
If possible I would change the signature to `create_vector(container, /, kind)`, where "/" is to disallow people from using a kwarg for the first input. The rationale is that the name of the first input changed, and arguably container is uglier (less expressive) than L